libs/blocks/card/cardUtils.js

```diff
@@ -36,10 +36,17 @@ export const addFooter = (links, container, merch) => {
   });
 };
 
-export const addWrapper = (el, section, cardType) => {
+export const addWrapper = (el, cardType) => {
+  const fragment = el.closest('.fragment');
+  if (fragment) {
```

Comment:
suggestion to combine `fragment` and `fragmentSection` selectors.
`const fragmentSection = el.closest('.fragment')?.closest('.section');`

Reply:
Why not just `.closest('.section .fragment')`?

Reply:
i need fragment too, to set a flex display on it 
should I change it to something like
```
el.closest('.fragment')?.style.display = 'flex';
el.closest('.fragment .section')?.replaceWith(el);
```
?

it will be shorter but el.closest('.fragment') is then computed twice.

Reply:
@3ch023 if you need `fragment` as well keep it, I thought it wasn't used.

Reply:
Yeah, I just reacted to @yesil's comment, but if you need to set properties on both, then it should be fine. And yeah, order was not correct, just highlighting the other possible approach.